www_src/pages/section-1/section-1.jsx

```diff
@@ -1,11 +1,18 @@
 var React = require('react');
 var render = require('../../lib/render.jsx');
+var Link = require('../../components/link/link.jsx');
 
 var Main = React.createClass({
   render: function () {
     return (
-      <div>
-        <div className="tile"><img src="../../img/demo.png" /></div>
+      <div className="demo">
+        <Link url="/map/123" href="/pages/map" className="tile tile-demo">
```

Comment:
The previous stubs have `<Link>` inside the "inner" div. Was using this different ordering intentional?